guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -122,15 +122,15 @@ _990_
 ifndef::foreman-deb[]
 ----
 # {package-install} nfs-utils
-# systemctl enable rpcbind nfs-server
-# systemctl start rpcbind nfs-server nfs-lock nfs-idmapd
+# systemctl enable --now rpcbind nfs-server
+# systemctl start nfs-lock nfs-idmapd
 ----
 endif::[]
 ifdef::foreman-deb[]
 ----
 # {package-install} nfs-kernel-server nfs-common
-# systemctl enable rpcbind nfs-server
-# systemctl start rpcbind nfs-server nfs-idmapd
+# systemctl enable --now rpcbind nfs-server
+# systemctl start nfs-idmapd
```

Comment:
It's weird that some services are started but not enabled, which means that it's broken after a reboot? Not sure what would be best.

At least on Fedora it looks like `nfs-server` only has `Wants` on `nfs-idmapd` but no `Requires`. Possible bug?

Reply:
This is Debian, which enables services by default. But I do agree, just put it to the `enable --now` line.